components/search/search.tsx

```diff
@@ -85,6 +88,30 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
         new SearchUserProvider(actions.autocompleteUsersInTeam),
     ]);
 
+    const isDesktop = isDesktopApp() && isServerVersionGreaterThanOrEqualTo(getDesktopVersion(), '4.7.0');
+    if (enableFindShortcut) {
+        const handleKeyDown = useCallback((e: KeyboardEvent) => {
+            if (Utils.cmdOrCtrlPressed(e) && Utils.isKeyPressed(e, Constants.KeyCodes.F)) {
+                if (isDesktop || (!isDesktop && e.shiftKey)) {
+                    if (hideSearchBar) {
+                        actions.openRHSSearch();
+                        setKeepInputFocused(true);
+                    }
+                    handleUpdateSearchTerms(`in:${currentChannel.name} `);
+                    handleFocus();
+                }
+            }
+        }, [currentChannel, hideSearchBar]);
```

Comment:
In some routes there's no currentChannel, such as global threads, and maybe others in the future

```suggestion
    useEffect(() => {
        if (!enableFindShortcut) {
            return undefined;
        }

        const handleKeyDown = (e: KeyboardEvent) => {
            if (Utils.cmdOrCtrlPressed(e) && Utils.isKeyPressed(e, Constants.KeyCodes.F)) {
                if (isDesktop || (!isDesktop && e.shiftKey)) {
                    if (hideSearchBar) {
                        actions.openRHSSearch();
                        setKeepInputFocused(true);
                    }
                    if (currentChannel) {
                        handleUpdateSearchTerms(`in:${currentChannel.name} `);
                    }
                    handleFocus();
                }
            }
        };

        document.addEventListener('keydown', handleKeyDown);
        return () => {
            document.removeEventListener('keydown', handleKeyDown);
        };
    }, [currentChannel, hideSearchBar]);
```